backend/src/models/v2/Release.ts

```diff
@@ -15,7 +17,7 @@ export interface ReleaseInterface {
   draft: boolean
 
   fileIds: Array<string>
-  images: Array<string>
+  images: Array<FlattenedModelImage>
```

Comment:
For similar things we've defined interfaces in the same file. See `CollaboratorEntry`, `AccessRequestMetadata`, `ReviewResponse ` for examples

Reply:
This has also already been defined https://github.com/gchq/Bailo/blob/709b0a4d477c71508a2394e0b5922a2b6d1a1880/backend/src/services/v2/registry.ts#L18-L22 
I would suggest moving the existing definition above into this file.
I would also suggest we rename `model` to be `name` which is how that field is referred to in the Docker docs. It also seems a little less confusing